lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1006,7 +1009,7 @@ def view_init(self, elev=None, azim=None, roll=None, vertical_axis="z"):
             The axis to align vertically. *azim* rotates about this axis.
         """
 
-        self.dist = 10
+        self.dist = 10  # The camera distance from origin. Behaves like zoom
```

Comment:
Can we make dist a private attribute? The user never interacts with it. I've made the change but I can revert if needed